src/app/cube/collection-details/collection-details.component.scss

```diff
@@ -1,6 +1,31 @@
 @import '~_vars.scss';
 $sidebar-width: 350px;
 $content-width: calc(#{$max-width} - #{$sidebar-width});
+
+img {
+    height: 100px;
+    margin-bottom: 30px;
+}
+
+.title__icon {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    text-align: center;
+    margin: auto;
+    margin-bottom: 20px;
+    background: white;
+    color: #4290ea;
```

Comment:
Why is this color hardcoded? Can we replace this with one of the existing SCSS variables?

Reply:
Updated to use color variable.